examples/exec.rs

```diff
@@ -0,0 +1,61 @@
+//! This example will run a non-interactive command inside the container using `docker exec`
+
+use bollard::container::{Config, RemoveContainerOptions};
+use bollard::Docker;
+
+use bollard::exec::{CreateExecOptions, StartExecResults};
+use futures_util::stream::StreamExt;
+
+const IMAGE: &'static str = "alpine:3";
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error + 'static>> {
+    let docker = Docker::connect_with_unix_defaults().unwrap();
+
+    let alpine_config = Config {
+        image: Some(IMAGE),
+        tty: Some(true),
+        ..Default::default()
+    };
+
+    let id = docker
+        .create_container::<&str, &str>(None, alpine_config)
```

Comment:
Can you please add a `docker.create_image(...` call before creating the container - that makes sure the image is pulled from the remote registry.

Reply:
Done